data/definitive-rubygems-authors-list.txt

```diff
@@ -0,0 +1,39 @@
+# This file has the definitive and declarative list of Samvera ruby
+# gems owners.  Each line must either be either an email address
+# of the associated Rubygems profile.
+aaron.collier@stanford.edu
+adam.wead@protonmail.com
+anna3lc@gmail.com
+armintor@gmail.com
+bess
+bmquinn
+cam156@psu.edu
+chris@cbeer.info
+cjcolvar
+dan.coughlin@gmail.com
+dchandekstark@gmail.com
+digger250@gmail.com
+drew@codhicitta.com
+dweber.consulting@gmail.com
+elrayle
+escowles@ticklefish.org
+glen.horton@gmail.com
+jenlindner@gmail.com
+jeremy.n.friesen@gmail.com
+jessie.keck@gmail.com
+johnson.tom@gmail.com
+jonathan@dnil.net
```

Comment:
hey @jrochkind , do you want keys to the samvera castle or nah

Reply:
Thanks! Same answer as jkeck! I find it useful to contribute to some gems in samvera we depend on from time to time, but if you are trying to clean things up feel free to get rid of me! (from rubygem owners; I guess from samvera github commit auth is another thing?)

Reply:
@jrochkind Same response from me, then: Nope, I think it's fine for you to retain access. Thought it prudent to ask first! Thanks, J-Rock. :)